client/src/components/CreateForm/CreateForm.scss

```diff
@@ -0,0 +1,175 @@
+@import "../../theme";
+
+.create-form-container {
+  padding-left: 22rem;
+
+  .h5 {
+    padding-bottom: 1rem;
+  }
+
+  .form-label {
+    font-weight: bold;
+  }
+
+  .text-control {
+    width: 25rem;
+  }
+
+  .text-control-small {
+    width: 10rem;
+  }
+
+  .select-control {
+    width: 10rem;
+  }
+
+  .multi-line-text-control {
+    width: 25rem;
+  }
+
+  .btn-toolbar {
+    padding-left: 5rem;
+    padding-bottom: 4rem;
+
+    .btn {
+      font-weight: bold;
+      text-transform: uppercase;
+      font-family: "Open Sans", sans-serif;
+    }
+  }
+
+  .advanced-information-link {
+    cursor: pointer;
+    color: $primary;
+    padding-top: 1.5rem;
+    padding-left: 8.5rem;
+    padding-bottom: 1rem;
+    font-weight: bold;
+    text-decoration: underline;
+  }
+
+  .address-coordinates-container {
+    border-style: solid;
+    border-color: lightgray;
+    border-width: thin;
+    padding: 0.5rem 0 0.5rem 0.5rem;
+    width: 30rem;
+    margin-bottom: 1rem;
+
+    span {
+      padding-bottom: 0.5rem;
+      display: flex;
+    }
+
+    .form-group {
+      padding-left: 1rem;
+    }
+  }
+
+  .date-container {
+    border-style: solid;
+    border-color: lightgray;
+    border-width: thin;
+    padding: 0.5rem 0 0.5rem 0.5rem;
+    width: 30rem;
+    margin-bottom: 1rem;
+
+    .form-row {
+      padding-left: 0.5rem;
+    }
+
+    .form-label {
+      display: flex;
+    }
+
+    .react-datepicker__input-container input {
+      color: #495057;
+      background-clip: padding-box;
+      border: 1px solid #ced4da;
+      border-radius: 0.25rem;
+      height: calc(1.5em + 0.75rem + 2px);
+      padding: 0.375rem 0.75rem;
+    }
+
+    .react-datepicker__day--selected {
+      background-color: $primary;
+    }
+  }
+
+  .fileUploader {
+    width: 25rem;
+
+    .fileContainer {
+      box-shadow: none;
+      margin: 0;
+      padding: 0;
+
+      .chooseFileButton {
+        background: $primary;
+        font-weight: bold;
+        text-transform: uppercase;
+        font-family: "Open Sans", sans-serif;
+        margin: 0 0 10px 50px;
+      }
+
+      .deleteImage {
+        background: $danger;
+      }
+    }
+
+    .errorMessage {
+      display: block;
+    }
+  }
+
+  .invalid-feedback {
+    font-size: medium;
+
+    &.materials {
+      display: block;
+    }
+  }
+
+  .tags-search {
+    width: 30rem;
+    min-height: 7rem;
+
+    &:focus-within {
+      border-color: #9ddcc0;
+      box-shadow: 0 0 0 0.2rem rgba(66, 184, 131, 0.25);
+    }
+
+    &:last-child {
+      margin-right: 0;
+    }
+
+    &.is-invalid {
+      border-color: #da6060;
+      background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' fill='%23da6060' viewBox='-2 -2 7 7'%3e%3cpath stroke='%23da6060' d='M0 0l3 3m0-3L0 3'/%3e%3ccircle r='.5'/%3e%3ccircle cx='3' r='.5'/%3e%3ccircle cy='3' r='.5'/%3e%3ccircle cx='3' cy='3' r='.5'/%3e%3c/svg%3E");
```

Comment:
Could you give a comment saying what this is an icon of or where you got it. Also by the way we have material icons loaded on the site, so if it has something that covers this you should use it https://material.io/resources/icons/?style=baseline

Reply:
This is actually stolen from the styling that `react-bootstrap` uses for invalid Form Controls. I had to modify the CSS a bit to get it to work with `TagsSearch` since it's not a true Form Control, hence this.